spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+      </p>
+
+      <emu-alg>
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. If _options_ is *undefined*, then
+          1. Let _options_ be ObjectCreate(*null*).
+        1. Else
+          1. Let _options_ be ? ToObject(_options_).
+        1. Let _t_ be ? GetOption(_options_, `"type"`, `"string"`, &laquo; `"cardinal"`, `"ordinal"` &raquo;, `"cardinal"`).
+        1. Set _pluralRules_.[[Type]] to _t_.
+        1. Let _opt_ be a new Record.
+        1. Let _matcher_ be ? GetOption(_options_, `"localeMatcher"`, `"string"`, &laquo; `"lookup"`, `"best fit"` &raquo;, `"best fit"`).
+        1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. Perform ? SetNumberFormatOptions(_pluralRules_, _options_, *0*, *3*).
+        1. Let _r_ be ResolveLocale(%PluralRules%.[[AvailableLocales]], _requestedLocales_, _opt_).
+        1. Set _pluralRules_.[[Locale]] to the value of _r_.[[Locale]].
+        1. Let _pluralCategories_ be a List of Strings representing the possible results of plural selection.
+        1. Set _pluralRules_.[[PluralCategories]] to CreateArrayFromList(_pluralCategories_).
+        1. Return _pluralRules_.
+      </emu-alg>
+
+    </emu-clause>
+
+    <emu-clause id="sec-getoperands" aoid="GetOperands">
+       <h1>GetOperands ( _s_ )</h1>
+       <p>
+         When the GetOperands abstract operation is called with argument _s_. It performs the following steps:
+       </p>
+
+       <emu-alg>
+         1. Assert: Type(_s_) is String.
+         1. Let _n_ be ? ToNumber(_s_).
+         1. Assert: _n_ is finite.
+         1. Let _dp_ be ? Call(%StringProto_indexOf%, _s_, &laquo; `"."` &raquo;).
+         1. If _dp_ = -1, then
+           1. Set _iv_ to _n_.
+           1. Let _f_ be 0.
+           1. Let _v_ be 0.
+         1. Else,
+           1. Let _iv_ be the substring of _s_ from position 0, inclusive, to position _dp_, exclusive.
+           1. Let _fv_ be the substring of _s_ from position _dp_, exclusive.
+           1. Let _f_ be ? ToNumber(_fv_).
+           1. Let _v_ be ? ToLength(? Get(_fv_, `"length"`)).
+         1. Let _i_ be ? abs(? ToNumber(_iv_)).
+         1. If _f_ ≠ 0, then
+           1. Let _ft_ be the value of _fv_ stripped of trailing `"0"`.
+           1. Let _w_ be ? ToLength(? Get(_ft_, `"length"`)).
+           1. Let _t_ be ? ToNumber(_ft_).
+         1. Else,
+           1. Let _w_ be 0.
+           1. Let _t_ be 0.
+         1. Return a new Record { [[Number]]: _n_, [[IntegerDigits]]: _i_, [[NumberOfFractionDigits]]: _v_, [[NumberOfFractionDigitsWithoutTrailing]]: _w_, [[FractionDigits]]: _f_, [[FractionDigitsWithoutTrailing]]: _t_ }.
+       </emu-alg>
+
+       <emu-table id="table-plural-operands">
+         <emu-caption>Internal Slots of Plural Rules Operands</emu-caption>
```

Comment:
The caption doesn't really match, the table describes the **fields** of the **record** returned by GetOperands and not any **internal slots**.

Records and fields: https://tc39.github.io/ecma262/#sec-list-and-record-specification-type
Internal slots: https://tc39.github.io/ecma262/#sec-object-internal-methods-and-internal-slots

Reply:
will ask for some clarification and maybe some description for the table.

Reply:
using "Plural Rules Operands Record Fields". @littledan do you have any opinion about this?